numba_dpex/examples/kernel/atomic_op.py

```diff
@@ -10,12 +10,14 @@
 @ndpx.kernel
 def atomic_reduction(a):
     idx = ndpx.get_global_id(0)
```

Comment:
Please add some docstring explaining what the example does. It should be pointed out to users the memory order (relaxed) and the scope (device) of the atomic operation.